pipelines/pull_nhsn.R

```diff
@@ -0,0 +1,66 @@
+script_packages <- c(
+  "forecasttools",
+  "tidyverse",
```

Comment:
```suggestion
  "dplyr",
  "readr",
```
Prefer not to load the whole tidyverse but rather just the packages used